src/reports/client/main.cpp

```diff
@@ -0,0 +1,198 @@
+#include <iostream>
+#include <vector>
+#include <chrono>
+#include <thread>
+#include <cmath>
+
+#ifdef HAVE_MPI
+#include <mpi.h>
+#endif
+
+#include <bbp/reports/records.hpp>
+
+struct Neuron {
+    // some properties
+    int gid;
```

Comment:
`gid` isn't a concept in sonata, so this will be confusing, I think.  Is this the same thing as a `node_ids`

Reply:
Thats true, maybe I should change gid by node_id and compartments by elements?